x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -44753,8 +44753,6 @@
     "xpack.metricsData.assetDetails.metricsCharts.metric.label.read": "Lire",
     "xpack.metricsData.assetDetails.metricsCharts.metric.label.used": "Utilisé",
     "xpack.metricsData.assetDetails.metricsCharts.metric.label.write": "Écrire",
-    "xpack.metricsData.assetDetails.metricsCharts.network.label.rx": "Entrant (RX)",
```

Comment:
Those labels are still used in[ x-pack/plugins/observability_solution/metrics_data_access/common/inventory_models/shared/charts/constants.ts ](https://github.com/elastic/kibana/pull/184245/files#diff-1c17cb00f71ad818c37029d6fc954facacb27b73f9c344b61cdb50df57cce705)or we have new labels there 🤔

Reply:
Mmmh I don't recall to delete those, I will check

Reply:
I renamed the labels when I added them to the constants file, that's why was deleted, I improved the name. So they're going to update again. Thanks for pointing this out

Reply:
Thanks for the change!